tracer_test.go

```diff
@@ -93,7 +93,7 @@ func (s *tracerSuite) TestBeginRootSpan() {
 	s.NotNil(ss.duration)
 
 	s.metricsFactory.AssertCounterMetrics(s.T(), []metricstest.ExpectedMetric{
-		{Name: "jaeger.tracer.finished_spans", Value: 1},
+		{Name: "jaeger.tracer.finished_spans", Tags: map[string]string{"sampled": "y"}, Value: 1},
```

Comment:
I think we may want to tighten these tests by also asserting the length of these metrics to verify that nothing additional is emitted.

Reply:
I think that'd only make the tests more fragile. The objective here is to test that some specific methods are emitted.